### PR TITLE
fix(number): fix number format

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
Jira Ticket: [COV-802](https://occ-data.atlassian.net/browse/COV-802)
- [ ] Fix number format issue:

### CODE REVIEW

**Review by m0nhawk — DISMISSED** [review]

File: covid19-etl/etl/ctp.py [hunk 1]
```diff
@@ -324,8 +324,11 @@ def parse_row(self, row):
 
         for k, v in map_csv_fields.items():
             value = row[self.header_to_column[v]]
-            if value and value.lower() not in ["nan", "n/a"]:
-                summary_clinical[k] = int(value.replace(",", ""))
+            if value:
+                value = value.replace(",", "")
+                number = re.match(r"^([\s\d]+)$", value)
+                if number and value.lower() not in ["nan", "n/a"]:
```
Comment (paulineribeyre) [inline covid19-etl/etl/ctp.py:330]:
```suggestion
                if number:
```
"nan"/"n/a" would match the number regex

File: covid19-etl/etl/ctp.py [hunk 2]
```diff
@@ -324,8 +324,11 @@ def parse_row(self, row):
 
         for k, v in map_csv_fields.items():
             value = row[self.header_to_column[v]]
-            if value and value.lower() not in ["nan", "n/a"]:
-                summary_clinical[k] = int(value.replace(",", ""))
+            if value:
+                value = value.replace(",", "")
+                number = re.match(r"^([\s\d]+)$", value)
```
Comment (paulineribeyre) [inline covid19-etl/etl/ctp.py:329]: why \s?

Comment (paulineribeyre) [inline covid19-etl/etl/ctp.py:329]: or `try: int(value); except: pass`

Comment (thanh-nguyen-dang) [reply covid19-etl/etl/ctp.py:329]: try to match space sometime between numbers

Comment (thanh-nguyen-dang) [reply covid19-etl/etl/ctp.py:329]: except pass is kind of temporary code

Comment (paulineribeyre) [reply covid19-etl/etl/ctp.py:329]: but it matches weird stuff like `0 1 2` 🤔

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**